VM/src/lstrlib.cpp

```diff
@@ -1032,6 +1034,26 @@ static int str_format(lua_State* L)
                     break;
                 }
             }
+            case '*':
+            {
+                if (!FFlag::LuauTostringFormatSpecifier)
+                {
+                    luaL_error(L, "invalid option '%%*' to 'format'");
```

Comment:
Random bikeshed: Would it be fine for this to give a bespoke error message for benefit of consumers, or act as it does now where it acts the same as it would before?

Reply:
Generally we keep things consistent with what it looked like before the flag, so I'd keep as is.